androguard/core/bytecodes/apk.py

```diff
@@ -804,39 +796,63 @@ def _format_value(self, value):
                     value = self.package + "." + value
         return value
 
-    def get_element(self, tag_name, attribute, **attribute_filter):
-        """
-        Return element in xml files which match with the tag name and the specific attribute
-
-        :param tag_name: specify the tag name
-        :type tag_name: string
-        :param attribute: specify the attribute
-        :type attribute: string
-
-        :rtype: string
-        """
-        for i in self.xml:
-            if self.xml[i] is None:
-                continue
-            tag = self.xml[i].findall('.//' + tag_name)
-            if len(tag) == 0:
-                return None
-            for item in tag:
-                skip_this_item = False
-                for attr, val in list(attribute_filter.items()):
-                    attr_val = item.get(self._ns(attr))
-                    if attr_val != val:
-                        skip_this_item = True
-                        break
-
-                if skip_this_item:
-                    continue
-
-                value = item.get(self._ns(attribute))
-
-                if value is not None:
-                    return value
-        return None
+    def get_all_attribute_value(
```

Comment:
can you please add a docstring for the new methods?

Reply:
yes, this is the next thing left